### DISCUSSION TITLE
About @apostrophecms/html widget

### DISCUSSION THREAD

**Discussion by developer-sachidanand** [apostrophecms/apostrophe#3448] [Q&A]
Adding '@apostrophecms/html' widget in the field of default page doesn't renders it 

Any help will be greatly appreciated.

Thanks

**Comment by abea** [comment]
I have tested the HTML widget on a default page successfully. Please share some code if you would like help debugging. How is your area configured? What does the area tag in the template look like? Are there any errors in the browser console? Have you confirmed the embed code works outside of Apostrophe?
